prov/net/src/xnet_domain.c

```diff
@@ -133,6 +133,32 @@ static struct fi_ops_domain xnet_domain_ops = {
 	.query_collective = fi_no_query_collective,
 };
 
+static int xnet_update_wait_eq_list(struct xnet_domain *domain, bool add)
+{
+	struct xnet_fabric *fabric;
+	struct xnet_eq *eq;
+	struct dlist_entry *item;
+	int ret = 0;
+
+	fabric = container_of(domain->util_domain.fabric,
+			      struct xnet_fabric,
+			      util_fabric.fabric_fid);
+
+	ofi_mutex_lock(&fabric->util_fabric.lock);
+	dlist_foreach(&fabric->wait_eq_list, item) {
+		eq = container_of(item, struct xnet_eq, wait_eq_entry);
+		if (add)
+		    ret = xnet_eq_add_progress(eq, &domain->progress,
+					       &domain->util_domain.domain_fid);
+		else
+		    ret = xnet_eq_del_progress(eq, &domain->progress);
+		if (ret)
+		    break;
```

Comment:
spaces in place of tabs before ret and break calls

Reply:
Done